### DISCUSSION TITLE
%E4%B8%AD%E9%97%B4%E4%BB%B6/kafka/%E6%A0%B8%E5%BF%83%E7%9F%A5%E8%AF%86%E7%82%B9/kafka%E6%98%AF%E4%BB%80%E4%B9%88%EF%BC%9F%E6%9E%B6%E6%9E%84%E6%98%AF%E6%80%8E%E4%B9%88%E6%A0%B7%E7%9A%84%EF%BC%9F

### DISCUSSION THREAD

**Discussion by giscus** [xiaobaiTech/golang-full-stack#34] [Announcements]
# %E4%B8%AD%E9%97%B4%E4%BB%B6/kafka/%E6%A0%B8%E5%BF%83%E7%9F%A5%E8%AF%86%E7%82%B9/kafka%E6%98%AF%E4%BB%80%E4%B9%88%EF%BC%9F%E6%9E%B6%E6%9E%84%E6%98%AF%E6%80%8E%E4%B9%88%E6%A0%B7%E7%9A%84%EF%BC%9F

Kafka 是什么？ 后端开发八股文刷题指南 你是一个程序员，假设你维护了两个服务 A 和 B。 B 服务每秒只能处理 100 个消息，但 A 服务却每秒发出 200 个消息，B 服务哪里顶得住，分分钟被压垮。 那么问题就来了，有没有办法让 B 在不被压垮的同时，还能处理掉 A 的消息？ 当然有，没有什么是加一层中间层不能解决的，如果有，那就再加一层。...

https://golangguide.top/%E4%B8%AD%E9%97%B4%E4%BB%B6/kafka/%E6%A0%B8%E5%BF%83%E7%9F%A5%E8%AF%86%E7%82%B9/kafka%E6%98%AF%E4%BB%80%E4%B9%88%EF%BC%9F%E6%9E%B6%E6%9E%84%E6%98%AF%E6%80%8E%E4%B9%88%E6%A0%B7%E7%9A%84%EF%BC%9F.html

**Comment by Yinheng-Li** [comment]
写得很棒，感谢分享！顺便捉个虫，最后「总结」第一段中“像消息队列投递消息的是生产者”的“像”应该是“向”~

**Comment by Fat7Cat** [comment]
超棒。感谢你的无私分享。

**Comment by GuoweiTops** [comment]
感谢分享！
